src/lib.rs

```diff
@@ -174,6 +173,10 @@ pub use error::Error;
 mod sftp;
 pub use sftp::{Mode, RemoteFile, Sftp};
 
+#[cfg(docsrs)]
+/// Changelog for this crate, only present in doc.
+pub mod changelog;
```

Comment:
This is an interesting way to do changelogs! I generally prefer the [keep a changelog](https://keepachangelog.com/en/1.0.0/) format though, since it's fairly standard. Could we do that in a separate PR?

Reply:
Sure, I have removed the module `changelog` and will create a separate PR for it.